src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/FakeDiagnosticListenerObserver.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
```

Comment:
That's a good point. I will remove the winhttphandler functional tests since, as you say, that exact same code path is exercised by the httclient functional tests.